views/components/Graphs/BarGraph.jsx

```diff
@@ -23,7 +23,8 @@ const BarGraph = ({ graph }) => {
       containerComponent={
         <VictoryVoronoiContainer
           labels={({ datum: { study, studyTarget, count, valueLabel } }) => {
-            const showToolTip = study && count && study !== TOTALS_STUDY
+            const showToolTip =
+              study && count && study !== constants.TOTALS_STUDY
             return valueLabel !== 'N/A' && showToolTip
```

Comment:
should the check on valueLabel be part of `showToolTip`?
```suggestion
            const showToolTip =
              study && count && study !== constants.TOTALS_STUDY && valueLabel !== 'N/A'
            return showToolTip //...
```

Reply:
I think `'N/A/'` could be moved to a constant as well in this example 😄

Reply:
https://github.com/AMP-SCZ/dpdash/pull/147/commits/04f13bd1ffa2be4dabbec2ecb9bb841fb2abafa3

Reply:
looks great ☝️
🎱 (very optional nit) could add a newline between the definition of `showToolTIp` and the return statement 🎱